### DISCUSSION TITLE
No autosave after killing Diablo and end the game?

### DISCUSSION THREAD

**Discussion by johndoe71rus** [diasurgical/DevilutionX#1338] [Q&A] [answered]
No autosave after killing Diablo and end the game?

Reproduce:
1 make save
2 change equip
3 go down and kill Diablo without any save
4 start new game 
result
save only stuff on hero and backpack from save point 1

**Comment by AJenbo** [comment, marked as answer]
This is how the game works (originally). There are reasons you might not want to save after killing diablo as you cannot return to pick up your equipment or revisit other parts like the Hellfire levels.

If you want to have the save include the XP for killing diablo simply save as he is bleeding before the movie starts.

**Reply by aspiringautisticcoder** [reply]
I am a hoarder who keeps lots of stuff in town so I would hate to like beat Diablo and lose access to my gear and gold in town. lol.

Back in 1996/1997 when Diablo was out, New Game+ was a fairly rare thing in western RPGs I believe. Some Japanese RPGs back then like Chrono Trigger had New Game+ (which came over a year before Diablo on the SNES). And then Western RPGs started implementing this kind of end game feature. Single player Diablo was meant to be finished when you slay Diablo. The Nightmare and Hell difficulty levels aren't even accessible in Vanilla Diablo without the multiplayer exploit or mods like this.

If you want to create a new game with your character after beating Diablo (whether it be trying Nightmare or Hell. Power-levelling to 50, gold farming for elixirs and shopping runs, doing Laz runs, etc.) what you have to do is sell most of your stuff for gold and buy elixirs and books (sometimes there's something really good at Griswold/Adria/Wirt for purchase that you'd like to carry over to a next playthrough though) with your gold so that you can condense your non-equipped possessions to 40 inventory squares (+ 8 belt squares) as efficiently as possible. And then beat Diablo and create a new game.

**Comment by johndoe71rus** [comment]
> If you want to have the save include the XP for killing diablo simply save as he is bleeding before the movie starts.

Only now I found out that I lost experience from killing 4 Diablo

**Reply by AJenbo** [reply]
You get negative XP when killing Diablo?

**Comment by johndoe71rus** [comment]
no.
I made a save.
Killed several monsters and Diablo at level 16.
But didn't save when he bleeds.
Therefore, when I next started a new game, I had experience from saving, without experience for monsters and diablo

**Reply by qndel** [reply]
That's .... how the game works lol

**Reply by aspiringautisticcoder** [reply]
When I beat Diablo on Hell difficulty at clvl 37, I got 431,191 experience points. Which is a lot admittedly for a kill. But in the grand scheme of things, you need 22,164,762 experience points to get a level up at that point anyways so it's not a big deal. You can get 431,191 experience relatively quickly fighting (make sure to stone curse/tele-kill mages/succubi if you are a warrior. Stone curse with sorcerer versus triple immunes) in Hell/Hell. That's less than 2% of a level up at clvl 37.

Even in normal difficulty @ clvl 25, Diablo paid out only 47,499 experience points. That's 3.7% of a level up at clvl 25 as you need 1,293,323 experience points.

And if you are trying to power-level to 50, these lost experience points from Diablo are going to seem trivial.
